OpenStack-Rabbit-Consumer/usr/local/bin/aq_api.py

```diff
@@ -32,9 +40,32 @@ def verify_kerberos_ticket():
     logger.info("Kerberos ticket success")
     return True
 
+def setup_requests(url, method, desc):
+    verify_kerberos_ticket()
 
-def aq_make(hostname, personality=None, osversion=None, archetype=None, osname=None):
-    logger.info("Attempting to make templates for " + hostname)
+    s = requests.Session()
```

Comment:
single letter variables like `s` should really only be used with very small functions or as loop variables